### PR TITLE
Envelope labels

### PR DESCRIPTION
**Pull request by frenzymadness**
Nová možnost tisknout štítky na obálky.

### CODE REVIEW

File: registry/donor/forms.py [hunk 1]
```diff
@@ -103,3 +109,21 @@ def get_field_data(self):
...
+    def validate(self):
+        self.medal = Medals.query.get(self.medal_id.data)
+
+        if self.medal is None:
+            flash("Odeslána nevalidní data.", "danger")
+            return False
+
+        if not (0 <= self.skip.data < 16):
+            flash("Vynechat lze 0 až 15 štítků.", "danger")
+            return False
```
Comment (adelpopelkova) [inline registry/donor/forms.py:130]: I tested this manually and the propper message does not seem to show itself. I see only an info message in English.

**Review by adelpopelkova — COMMENTED** [review]
It seems to work well. I did a bit of manual testing. Here are a few findings:
- One tiny detail: the buttons seem to be too close to each other (this doesn't affect the functionality).
- Then the error message when entering a higher/lower number does not seem to be in Czech. 
- With the _Skipping first x number of labels_ maybe preparing the system for the situation when nothing is added (an empty field is sent). At the moment you get an error.
- With the labels sheet itself - is it planned to print out the whole page on the pre-cut labels? And does this layout fit the labels?

**Comment by frenzymadness** [comment]
>     * Then the error message when entering a higher/lower number does not seem to be in Czech.
Those are messages produced by the browser, I believe. I will test it today on a system with Czech locales.
>     * With the _Skipping first x number of labels_ maybe preparing the system for the situation when nothing is added (an empty field is sent). At the moment you get an error.
Good catch, will use 0 in that case.
>     * With the labels sheet itself - is it planned to print out the whole page on the pre-cut labels? And does this layout fit the labels?
Yes, exactly, I'll add a comment to the top od the HTML page.

File: registry/donor/forms.py (hunk 1 above)
Comment (frenzymadness) [reply registry/donor/forms.py:130]: It works if you have disabled javascript. If you use JS, the browser doesn't allow you to send the request to the server.

**Pull request merged**